__tests__/app.test.js

```diff
@@ -174,3 +174,59 @@ describe('GET /api/articles/:article_id/comments', () => {
         })
     })
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('201: inserts a new comment into the specified article, and returns that newly added comment', () => {
+        const commentToAdd = {
+            body: 'this is a bunch of nonsense',
+            author: "butter_bridge",
+            votes: 0,
+            created_at: "2020-10-16T11:28:00.000Z"
+        }
```

Comment:
You only need to send the author and body.

Why would created_at and votes exist at this point ?